### PR TITLE
Update rules-dsl.md

### PR DESCRIPTION
**Pull request by JensHauser**
Learnings from conversation with Rich added:
https://community.openhab.org/t/ecma-rule-to-trigger-after-system-fully-functional/152091/6

Signed-off-by: Jens Hauser<jens@hauser-digital.de>

### CODE REVIEW

File: configuration/rules-dsl.md [hunk 1]
```diff
@@ -241,6 +241,10 @@ In openHAB version 3 the System-based Trigger for startlevel had been added, val
 
 Startlevels less than 40 are not available as triggers because the rule engine needs to start up first before it can execute any rules.
 
+A rule triggering for a startlevel and being saved during development, will be reloaded.
```
Comment (stefan-hoehn) [inline configuration/rules-dsl.md:244]: A rule that triggers for a start level and is saved during development is reloaded.

File: configuration/rules-dsl.md [hunk 2]
```diff
@@ -241,6 +241,10 @@ In openHAB version 3 the System-based Trigger for startlevel had been added, val
...
+If the expected startlevel (or higher) has already been reached, this rule will trigger again.
+Writing rules with quite low startlevels might be too early, as for instance not all things are online yet.
```
Comment (stefan-hoehn) [inline configuration/rules-dsl.md:246]: This quite generic and more or less stating the obvious. Can you provide an example here to make it more tangible?

**Review by stefan-hoehn — CHANGES_REQUESTED** [review]
Many thanks for the contribution, Jens. See my comments.

**Comment by JensHauser** [comment]
Hi Stefan, hope, it fits now. Happy new year soon :-)

**Review by stefan-hoehn — APPROVED** [review]
Thanks for the contribution.

**Pull request merged**